assembly/assembly-main/src/assembly/bin/che.sh

```diff
@@ -286,321 +222,130 @@ set_environment_variables () {
-                   `OWNER READ:  ${OWNER_READ} \n`
-                   `OWNER WRITE: ${OWNER_WRITE} \n`
-                   `GROUP READ:  ${GROUP_READ} \n`
-                   `GROUP WRITE: ${GROUP_WRITE} \n`
-                   `OTHER READ:  ${OTHER_READ} \n`
-                   `OTHER WRITE: ${OTHER_WRITE} \n`
-                   `Run 'sudo chmod 666 /var/run/docker.sock' to give the right permissions."
-        return
-      fi
-
-      # CHE-1202: Improve error messages in case of docker ps failure
-      # Verify that docker client and server versions match
-      DOCKER_SERVER_VERSION=$(docker version --format '{{.Server.Version}}')
-      DOCKER_CLIENT_VERSION=$(docker version --format '{{.Client.Version}}')
-
-      if [[ "$DOCKER_SERVER_VERSION" != "$DOCKER_CLIENT_VERSION" ]]; then
-        error_exit "Running 'docker' succeeded, but 'docker ps' failed. \n`
-                   `The docker client version does not match the docker server version. \n`
-                   `DOCKER SERVER: ${DOCKER_SERVER_VERSION} \n`
-                   `DOCKER CLIENT: ${DOCKER_CLIENT_VERSION} \n`
-                   `This can occur if you are running Che as a container itself. \n`
-                   `The Che container has an internal docker client that uses your host's docker server. \n` 
-                   `Consider updating docker engine to have the versions match."
-        return
-      fi
-      
-      error_exit "Running 'docker' succeeded, but 'docker ps' failed. \n`
-                 `/var/run/docker.sock is ok and your docker client and server have matching versions. \n`
-                 `Run 'docker ps' and inspect the output for additional clues."
-      return
+  if [ "${HOST}" == "windows" ]; then
+    if [ -z ${DOCKER_HOST+x} ]; then
+      export DOCKER_HOST=tcp://localhost:2375
     fi
   fi
 
   # Hidden parameter
-  # Only used if running Che server inside a Docker container.
-  # Copies Che libraries to a temporary directory which is mounted by the container to be reachable by external host.
-  # Files must be copied otherwise host will overwrite them to blank.
-  if [ "${COPY_LIB}" == "true" ]; then
-    sudo chown -R user:user ${CHE_HOME}
-    rm -rf ${CHE_HOME}/lib-copy/*
-    mkdir -p ${CHE_HOME}/lib-copy
-    cp -rf ${CHE_HOME}/lib/* ${CHE_HOME}/lib-copy
-
-    export JAVA_OPTS="${JAVA_OPTS} -Dche.docker.che_host_network=bridge"
-  fi 
+  # Only used if this script is starting a Che Tomcat from inside a Docker container.
+  # Copies Che ws-agent and terminal to directory mounted by ws containers.
+  # The directory has to be on the host, not inside the container.
+  # The files are copied from within the container to a folder host-mounted.
+#  if [ "${CHE_IN_CONTAINER}" == "true" ]; then
+
+    # Make sure the user named "user" is the owner of the CHE_HOME directory.
+    # Make sure the user named "user" is the owner of the CHE_DATA directory.
```

Comment:
This is removed in che.sh and it's been added into docker.sh with an improved message.